frontend/mock-api/stored-queries/25.json

```diff
@@ -2,7 +2,20 @@
   "id": 25,
   "status": "DONE",
   "numberOfResults": 5,
-  "resultUrls": ["/api/results/1234-123123123-123123-1231.csv"],
+  "resultUrls": [
+    {
+      "label": "Alle Dateien",
+      "url": "http://localhost:8080/api/results/all.zip"
+    },
+    {
+      "label": "Bericht.pdf",
```

Comment:
So sometimes it's PDF, XLSX, but other times it's an actual label like this? Would it make sense to model this as 
```
{
  label: string;
  url: string;
  ending: string;
}
```

did you discuss something like that?

Reply:
I don't think that it we should do it like that, since we are already transferring the "real" ending as the url ending. So we would only be transferring redundant data.

Reply:
Okay, so are these three examples common for a `stored query`? If I understood correctly, the editor would always yield a CSV, right? And a form query result would yield these pdf / xlsx / other, right?

So we should use the CSV / XLSX examples here, no?